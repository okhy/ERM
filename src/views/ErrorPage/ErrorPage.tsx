import * as React from "react";

const ErrorPage = () => <span>404</span>;

export default ErrorPage;
