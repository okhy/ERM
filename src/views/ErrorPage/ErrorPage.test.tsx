import * as React from "react";
import { shallow } from "enzyme";
import ErrorPage from "./ErrorPage";

describe("ErrorPage ...", () => {
  it("... renders properly", () => {
    const wrapper = shallow(<ErrorPage />);
    expect(wrapper.find(ErrorPage)).toBeTruthy();
  });
  it("... matches snapshot", () => {
    const wrapper = shallow(<ErrorPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
