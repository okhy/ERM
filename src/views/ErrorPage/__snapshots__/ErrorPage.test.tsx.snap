// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage ... ... matches snapshot 1`] = `ShallowWrapper {}`;
